packages/pigeon/tool/run_tests.dart

```diff
@@ -184,6 +184,7 @@ Future<void> main(List<String> args) async {
     androidJavaUnitTests,
     androidJavaLint,
     androidKotlinUnitTests,
+    androidKotlinLint,
```

Comment:
This will add it to CI, but it should also be added [here](https://github.com/flutter/packages/blob/322f3ed357a606d4b70d596ec3ad84d5cabef8f2/packages/pigeon/tool/test.dart#L80) for local runs of all tests.